modules/rosetta/src/test/scala/org/tessellation/rosetta/domain/construction/ConstructionServiceSuite.scala

```diff
@@ -36,6 +39,53 @@ object ConstructionServiceSuite extends MutableIOSuite with Checkers with Transa
       .forAsync[IO](sharedKryoRegistrar)
       .flatMap(kryo => SecurityProvider.forAsync[IO].map(securityProvider => (securityProvider, kryo)))
 
+  def generateTestTransactions(wantSignedTransaction: Boolean) =
+    sharedResource.use { res =>
```

Comment:
instead of `sharedResource.use` you should specify required implicits:
```scala
def generateTestTransactions(wantSignedTransaction: Boolean)(implicit val S: SecurityProvider[IO], K: KryoSerializer[IO]) = ...
```

Reply:
Changed.